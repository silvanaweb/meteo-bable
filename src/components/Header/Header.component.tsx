import React from "react";
import "./Header.css";

const Header: React.FC = () => (
  <header className="Header text-align--center">
    <div className="content-container">Meteo Bable</div>
  </header>
);

export { Header };
