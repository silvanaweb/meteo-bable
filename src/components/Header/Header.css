.Header {
  background-color: var(--secondary-color);
  color: var(--primary-color);
  font-size: 3rem;
  font-weight: bold;
  line-height: 1;
  padding: var(--s-size) 0;
}
