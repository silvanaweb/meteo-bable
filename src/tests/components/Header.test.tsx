import { shallow, ShallowWrapper } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";
import { Header } from "../../components/Header/Header.component";

let wrapper: ShallowWrapper;

beforeEach(() => {
  wrapper = shallow(<Header />);
});

test("should render Header correctly", () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});
