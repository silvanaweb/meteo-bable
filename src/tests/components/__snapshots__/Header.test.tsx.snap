// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header
  className="Header text-align--center"
>
  <div
    className="content-container"
  >
    Meteo Bable
  </div>
</header>
`;
